protocol/avdl/keybase1/teams.avdl

```diff
@@ -90,18 +90,22 @@ protocol teams {
     map<TeamInviteID,AnnotatedTeamInvite> annotatedActiveInvites;
   }
 
+  record UserVersion {
+    UID uid;
+    Seqno eldestSeqno;
+  }
+
+  @typedef("string")
+  record UserVersionPercentForm {}
```

Comment:
are there more places we can thread this type that are currently using strings?  my guess is that other places might benefit from this type too...  maybe not?

Reply:
seems like compiler should have found them all since I changed the type that PercentForm() returns to use this.